shared/secret_manager_cloud_cloud_test.go

```diff
@@ -0,0 +1,37 @@
+// +build cloud
+
+package shared
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+/*
+	TestCloudSecretManagerGetSecret get the secrets on a real Google Cloud
+	Secret Manager because there is no emulator like Datastore.
+
+	If it needs to be setup again, run the following:
+	gcloud secrets create test-secret --replication-policy="automatic"
+	echo -n "test-secret-value" | gcloud secrets versions add test-secret --data-file=-
+	gcloud secrets add-iam-policy-binding test-secret --member='serviceAccount:github-cicd@wptdashboard-staging.iam.gserviceaccount.com' --role='roles/secretmanager.secretAccessor'
+*/
+func TestCloudSecretManagerGetSecret(t *testing.T) {
+	require.NotEmpty(t, runtimeIdentity.AppID, "Unable to find project ID")
```

Comment:
Should a fake `runtimeIdentity.AppID` env be passed in the Makefile? Or this has to be the real wptdashboard-staging AppID?

Reply:
great question! it has to be the real wptdashboard-staging [AppID](https://github.com/web-platform-tests/wpt.fyi/pull/3016/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R94) because that project has the test secret manager value.